packages/cli/src/cog/cutline.ts

```diff
@@ -155,7 +155,7 @@ export class Cutline {
    * Generate an optimized WebMercator tile cover for the supplied source images
    * @param sourceMetadata contains images bounds and projection info
    */
-  optimizeCovering(sourceMetadata: SourceMetadata): NamedBounds[] {
+  optimizeCovering(sourceMetadata: SourceMetadata, maxImageSize?: number): NamedBounds[] {
```

Comment:
this can be defaulted in the function declaration with `, maxImageSize = MaxImagePIxelWidth` and means you can remove l168